elasticdl/python/tests/in_process_master.py

```diff
@@ -15,9 +17,15 @@ def ReportVariable(self, req):
         return self._m.ReportVariable(req, None)
 
     def ReportGradient(self, req):
+        for callback in self._callbacks:
+            if test_call_back.ON_REPORT_GRADIENT_BEGIN in callback.call_times:
+                callback()
         return self._m.ReportGradient(req, None)
 
     def ReportEvaluationMetrics(self, req):
+        for callback in self._callbacks:
+            if test_call_back.ON_REPORT_GRADIENT_BEGIN in callback.call_times:
```

Comment:
should be `ON_REPORT_EVALUATION_METRICS_BEGIN`?

Reply:
Fixed.